README.md

```diff
@@ -337,6 +337,9 @@ Collector exporter packages and types are renamed:
   - `CollectorExporterNodeBase` => `OTLPExporterNodeBase`
   - `CollectorMetricExporter` => `OTLPMetricExporter`
   - `CollectorTraceExporter` => `OTLPTraceExporter`
+- W3C propagators in @opentelemetry/core were renamed
+  - `HttpTraceContextPropagator` -> `W3CTraceContextPropagator`
+  - `W3CBaggagePropagator` -> `W3CBaggagePropagator`
```

Comment:
```suggestion
  - `HttpBaggagePropagator ` -> `W3CBaggagePropagator`
```

Reply:
Ah good catch. I think I made that original mistake.